admin/controllers/check-window.js

```diff
@@ -32,9 +35,39 @@ const controller = {
     req.breadcrumbs('Manage check windows', '/check-window/manage-check-windows')
     req.breadcrumbs(res.locals.pageTitle)
     return res.render('check-window/create-check-window', {
+      checkWindowData: {},
       breadcrumbs: req.breadcrumbs(),
+      error: new ValidationError(),
       currentYear: moment().format('YYYY')
     })
+  },
+
+  /**
+   * Submit check window
+   * @param req
+   * @param res
+   * @param next
+   * @returns {Promise.<void>}
+   */
+  submitCheckWindow: async (req, res, next) => {
+    const requestData = req.body
+    try {
+      await checkWindowV2AddService.process(requestData)
```

Comment:
why is this method called `process`?  is there something more indicative of its purpose that you can choose?

Reply:
@GuyHarwood  this service method:
- does validate the data and
- (on a later PR) will call a submit service method that will process them into an acceptable format for the data service
Would `validateAndSubmit` work better than `process`?

Reply:
submit is fine

Reply:
@GuyHarwood updated with submit